docs/photoniq/prerendering/prerendering-settings.md

```diff
@@ -95,3 +104,9 @@ In the web console, this displays the origin allowlist value if one is set. For
 In the web console, this displays current header keys and values. For more information about headers, refer to [Prerendering Headers](prerendering-headers).
 
 ![Add Response Header](/img/prerendering/add-response-header.png)
+
+### Pass-through Headers
+
+You can specify header values from a bot's request to be sent directly to the Prerender service. Click **Edit** to update the list.
```

Comment:
I think `specify headers` is better than `specify header values`, because the header (key/value) is forward, not only the value.

Reply:
Updated